lib/backends/fs.js

```diff
@@ -1,212 +1,211 @@
 /* istanbul ignore file */
 
-var glob = require('glob').glob,
-    yaml = require('js-yaml'),
-    path = require('path'),
-    fs = require('fs'),
-    callsites = require('callsites'),
-    findup = require('findup'),
-    deepCloneMerge = require('deep-clone-merge'),
-    async = require('async');
+let glob = require('glob').glob;
+let yaml = require('js-yaml');
+let path = require('path');
+let fs = require('fs');
+let callsites = require('callsites');
+let findup = require('find-up');
+let deepCloneMerge = require('deep-clone-merge');
+let async = require('async');
 
 function getcallsite(cb) {
-  var paths = callsites(),
-      root = path.resolve(__dirname, '../../');
-
-  // iterate over the call stack and find the first path that is not inside the i18n-future directory
-  var source = paths
-    .map(function (p) { return p.getFileName(); })
-    .reduce(function (src, p) {
-      if (src) {
-        return src;
-      } else if (path.resolve(root, p).indexOf(root) < 0) {
-        return p;
-      }
-    }, null);
+    let paths = callsites();
+    let root = path.resolve(__dirname, '../../');
+
+    // iterate over the call stack and find the first path that is not inside the i18n-future directory
+    let source = paths
+        .map(function (p) { return p.getFileName(); })
+        .reduce(function (src, p) {
+            if (src) {
+                return src;
+            } else if (path.resolve(root, p).indexOf(root) < 0) {
+                return p;
+            }
+        }, null);
 
     // if such a path is found, then findup from that path to a directory containing a package.json
     if (source) {
-      findup(source, 'package.json', cb);
+        findup(source, 'package.json', cb);
```

Comment:
`find-up` seems to have a different API to `find-up` where it doesn't accept a callback like this. Are we sure this is working?

Reply:
I think this might be more around the hot-reloading of changes to locales. Does that work okay?

Reply:
Hmm there's some branching logic in here so maybe we need to find some specific case that uses this. It's almost certainly not going to be working properly, whatever it is, at the moment because this function returns a Promise and we're not waiting for that to resolve or reject currently. Also the arguments you're passing in don't match up with the API.

Reply:
That line is also not covered by unit tests

Reply:
I think the easiest way to verify this is going to be adding some unit test coverage for it